presto-iceberg/src/main/java/io/prestosql/plugin/iceberg/IcebergConnector.java

```diff
@@ -71,7 +73,8 @@ public IcebergConnector(
             List<PropertyMetadata<?>> sessionProperties,
             List<PropertyMetadata<?>> schemaProperties,
             List<PropertyMetadata<?>> tableProperties,
-            ConnectorAccessControl accessControl)
+            ConnectorAccessControl accessControl,
+            IcebergProcedures procedures)
```

Comment:
Removed.  I added it because I couldn't figure out how to get the `Set<Procedure>` from an `Injector`, which is needed in `InternalIcebergConnectorFactory`.  I've since figured that out, and am glad to be able to remove the `IcebergProcedures` and `IcebergProceduresModule`.